Doc/library/pathlib.rst

```diff
@@ -1055,3 +1071,42 @@ call fails (for example because the path doesn't exist):
       'Text file contents'
 
    .. versionadded:: 3.5
+
+Correspondence to tools in the :mod:`os` package
+------------------------------------------------
+
+If you're more familiar with :mod:`os.path` module, here's a correspondence
+table on how the same things may be accomplished with :mod:`pathlib`.
+
+.. note::
+
+   Although :func:`os.path.relpath` and :meth:`PurePath.relative_to` have some
```

Comment:
I would simplify this note to:
```
Although :func:`os.path.relpath` and :meth:`PurePath.relative_to` have some overlapping use-cases, their semantics differ enough to warrant not considering them equivalent.
```

If people want to dive deeper they can then read the docs for the two functions.